src/MovieCards.jsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import './MovieCards.css'
+import { useStore } from "zustand";
+import { likedMoviesStore, searchStore } from "./App";
+
+
+export function MovieCard(props) {
+    // console.log("FROM movie card")
+    // console.log(props.data)
+    let [info, setInfo] = useState({})
+    let [liked, setLike] = useState(false)
+    
+
+    useEffect(() => {
+
+        (async () => { const url = `https://www.omdbapi.com/?i=` + props.data.imdbID + `&apikey=881763cb`
+            try {
+                const response = await fetch(url)
+                if (!response.ok) {
+                    throw new Error(`Response status: ${response.status}`)
+                }
+                info = await response.json()
+                //console.log(info)
+                setInfo(info)
+            } catch (error) {
+                console.error(error.message)
+            }})()
+    },[])
+
+
+    const addLike = likedMoviesStore((state) => state.addLike)
+    const removeLike = likedMoviesStore((state) => state.removeLike)
+    
+
+    function handleLike(movie) {
+        // console.log(movie)
+        if (!movie.fav) {
+            addLike(movie)
+        }else {
+            removeLike(movie)
+        }
+    }
+
+
+        return<>
+            <div class="movie_card" id="bright">
+                <div class="info_section">
+                    <div class="movimage">
+                        <img class="locandina" src={info.Poster} />
+                    </div>
+                    <div class="movinfo">
+
+                            <h1>{info.Title}</h1>
+                            
+                            <h4>{info.Year}, {info.Director}</h4>
+                            <span class="minutes">{info.Runtime}</span>
+                            <p class="type">{info.Genre}</p>
+                            <div class="movie_desc">
+                            <p class="text">
+                                {info.Plot}
+                            </p>
+
+                            </div>
+                            <div class="movsocial">
+                        <ul>
+                                <button onClick={() => handleLike(props.data)} className={props.data.fav ? "btn btn-danger btn-sm" : "btn btn-primary btn-sm"} id="likebutton">
```

Comment:
it's a good practice to destructure the props in a react component 
https://www.w3schools.com/react/react_es6_destructuring.asp